### PR TITLE
Abductors now know how to wipe memories of abduction

### PR DESCRIPTION
**Pull request by ghost**
:cl:
rscadd: Abductors finally figured out how to prevent abductees from screaming "HELP I WAS ABDUCTED" after they were experimented on.
/:cl:

### CODE REVIEW

**Comment by Jc1469** [comment]
Eeh.. I feel like this is more of a rules thing than a code thing personally.

**Comment by ghost** [comment]
i think it'd be a good code thing since it's not a rules thing

File: code/game/gamemodes/miniantags/abduction/machinery/experiment.dm [hunk 1]
```diff
@@ -186,6 +187,7 @@
 			H << "<B>Objective #[obj_count]</B>: [objective.explanation_text]"
 			obj_count++
 		ticker.mode.update_abductor_icons_added(H.mind)
+		H << "<span class='warning'><b>You can't remember how you got here or what was did to you..</b></span>"
```
Comment (Super3222) [inline code/game/gamemodes/miniantags/abduction/machinery/experiment.dm]: "or what was did to you"
please change did to done.

**Review by Super3222 — CHANGES_REQUESTED** [review]
Memes

**Comment by Slinky5890** [comment]
This was discussed a bit ago when we were making abductor rules, and decided that we would let the people remember their abduction.

File: code/game/gamemodes/miniantags/abduction/machinery/experiment.dm [hunk 2]
```diff
@@ -186,6 +187,7 @@
 			H << "<B>Objective #[obj_count]</B>: [objective.explanation_text]"
 			obj_count++
 		ticker.mode.update_abductor_icons_added(H.mind)
+		H << "<span class='warning'><b>You can't remember how you got here or what was done to you..</b></span>"
```
Comment (ShadowDeath6) [inline code/game/gamemodes/miniantags/abduction/machinery/experiment.dm]: ...

**Comment by ghost** [comment]
@Slinky5890 does it particularly matter? it could be useful.

**Comment by Mrcreepssss** [comment]
if you make it a special tool that could be good, i mean "ohshit i forgot the memory erasure they're coming for us" would make some good scenario's i imagine.

**Pull request merged**